client-misk/src/main/kotlin/app/cash/backfila/client/embedded/EmbeddedBackfilaModule.kt

```diff
@@ -0,0 +1,17 @@
+package app.cash.backfila.client.embedded
+
+import app.cash.backfila.client.BackfilaApi
+import app.cash.backfila.client.misk.BackfillModule
+import app.cash.backfila.client.internal.EmbeddedBackfila
+import com.google.inject.AbstractModule
+
+/**
+ * Use this to connect to an embedded Backfila implementation in development or testing. You will
+ * also need to install a [BackfillModule].
+ */
+class EmbeddedBackfilaModule : AbstractModule() {
```

Comment:
I was wondering about this. @swankjesse is there some standard KAbstractModule coming to Guice any time soon? Or should every project be defining its own copy of KAbstractModule, or just leave it this way?

Reply:
This is in progress but yeah we changed this so people didn't need the dependency here. Jesse might have better information